web/projectDetails.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/*
+ * Copyright (C) 2016 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define('PHPREPORT_ROOT', __DIR__ . '/../');
+
+/* We check authentication and authorization */
+require_once(PHPREPORT_ROOT . '/web/auth.php');
+
+/* Include the generic header and sidebar*/
+define('PAGE_TITLE', "PhpReport - Projects Summary");
+include_once("include/header.php");
+include_once("include/sidebar.php");
+include_once(PHPREPORT_ROOT . '/web/services/WebServicesFunctions.php');
+$user = $_SESSION['user'];
+?>
+
+<script type="text/javascript" src="js/include/ExportableGridPanel.js"></script>
+<script type="text/javascript">
+    var user = "<?php echo $user->getLogin(); ?>";
+</script>
```

Comment:
I think this parameter doesn't make sense. We want to get the lists of customers and projects regardless of the user. Could you try to remove it?